homeassistant/components/media_player/sonos.py

```diff
@@ -17,12 +19,34 @@
     SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET, SUPPORT_CLEAR_PLAYLIST,
     SUPPORT_SELECT_SOURCE, MediaPlayerDevice)
 from homeassistant.const import (
-    STATE_IDLE, STATE_PAUSED, STATE_PLAYING, STATE_UNKNOWN, STATE_OFF,
-    ATTR_ENTITY_ID)
+    STATE_IDLE, STATE_PAUSED, STATE_PLAYING, STATE_OFF, ATTR_ENTITY_ID)
 from homeassistant.config import load_yaml_config_file
 import homeassistant.helpers.config_validation as cv
+from homeassistant.util.async import run_coroutine_threadsafe
+
+REQUIREMENTS = ['SoCo==0.12']  # If you change this: see patch below
+
+
+_SOCO_PATCHED = False
+
+
+def _patch_soco():
```

Comment:
I don't like this. I see that your patch has been merged so I rather have them put out a new release or we start depending on a specific commit in their repo.